src/main/java/org/jitsi/jicofo/FocusManager.java

```diff
@@ -320,19 +320,45 @@ public boolean conferenceRequest(
         room = room.toLowerCase();
 
         JitsiMeetConferenceImpl conference;
+        String id;
         synchronized (conferencesSyncRoot)
         {
-            if (!conferences.containsKey(room))
+            conference = conferences.get(room);
+            if (conference == null)
+            {
+                id = generateConferenceId();
```

Comment:
is there a reason to do these things here and not in the same block where we call `createConference`? then all the 'create new conference'-type logic would be in the same place